README.md

```diff
@@ -1,6 +1,6 @@
 # Address Book (Level 4)
 
-[![Build Status](https://travis-ci.org/se-edu/addressbook-level4.svg?branch=master)](https://travis-ci.org/se-edu/addressbook-level4)
+[![Build Status](https://travis-ci.org/CS2103JAN2017-T09-B3/main.svg?branch=master)](https://travis-ci.org/se-edu/addressbook-level4)
```

Comment:
similar to this, you need to update the other badges (coverall, codacy etc.,) to reflect to your project.